tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -925,6 +925,47 @@ var _ = Describe("odo devfile push command tests", func() {
 		Context("odo push (without -f) is executed", func() {
 			ensureFilesSyncedTest(commonVar.Project, false)
 		})
+
+		When("node-js application is created and pushed with devfile schema 2.2.0", func() {
+			ensureResource := func(output, cpulimit, cpurequest, memoryrequest string) {
+				It("should push the component successfully and check cpuLimit,cpuRequests,memoryRequests", func() {
+					By("check for cpuLimit", func() {
+						podName := commonVar.CliRunner.GetRunningPodNameByComponent(cmpName, commonVar.Project)
+						output = helper.Cmd("oc", "get", "pods", podName, "-o", "jsonpath='{.spec.containers[0].resources.limits.cpu}'").ShouldPass().Out()
```

Comment:
Could you please use kubectl instead, or the tests will fail on Kubernetes cluster

Reply:
understood, updating